todo/src/components/TodoList.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import {connect} from 'react-redux'
+import {toggleComplete, removeTodo} from '../actions/actions.js'
+
+const TodoList = (props) =>{
+
+
+
+    if(!props.todos.length){
+        return null
```

Comment:
Could throw out a message here as well, but again great job considering the different states that your list could be in.